tests-requirements.txt

```diff
@@ -1 +1,2 @@
-nose
```

Comment:
There weren't any tests before, so I figured it'd be best to start with a clean slate. I'm going to document that the way to run the tests is ``python setup.py test`` which currently does not use nose. If that changes I'll add it back :)